__tests__/app.test.js

```diff
@@ -101,3 +101,33 @@ describe("/api/articles/:article_id", () => {
           });
     });
 });
+
+describe("/api/articles", () => {
+    test("GET 200: sends an array of article objects", () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response) => {
+            expect(response.body.articles.length).toBe(13);
+            response.body.articles.forEach((article) => {
+            expect(typeof article.article_id).toBe("number");
+            expect(typeof article.title).toBe("string");
+            expect(typeof article.topic).toBe("string");
+            expect(typeof article.author).toBe("string");
+            expect(typeof article.created_at).toBe("string");
+            expect(typeof article.votes).toBe("number");
+            expect(typeof article.article_img_url).toBe("string");
+            expect(typeof article.comment_count).toBe("string");
+            });
+        });
+    });
+
+    test("Ensure article object array is sorted by created_at date descending", () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response) => {
+            expect(response.body.articles[0].created_at).toBe("2020-11-03T09:12:00.000Z");
```

Comment:
you asserted the first one in the list is the one which is expected. What about the rest? This test does not give me confidence in the behaviour you are describing.